### DISCUSSION TITLE
Allow / (re-)Add 'initial' to UI Toggle creation

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#12944] [Dashboard]
As is stated in https://www.home-assistant.io/integrations/input_boolean/#configuration UI is now the preferred method for creating input_boolean (known as Toggle in the UI)

However, an important configuration option we have in Yaml, `initial:`, is not an option via the UI. As it isnt an attribute, we can not customize it either. 

Aware the booleans are restored, this is exactly the reason the option for 'initial' is so important. Sometimes we just need it to be fixed to either on or off at Homeassistant start, no matter the state is was in at restart.

Best example:

```
input_boolean:

  just_started:
    name: Just started
    initial: on
```

which is turned off after a certain amount of time and then remains Off. Dont want that at startup....

We could be writing an automation. for that possible, to set the booleans to the state at startup, but that seems like some serious overhead and honestly feels contrived for such a fundamental option on a core integration.

Lastly we could leave it all in YAML, but that would be against the preferred way...

So, please add the initial option to the UI creation for Toggles.

**Comment by KTibow** [comment]
Duplicate of #12846.
